stairway/src/main/java/bio/terra/stairway/queue/QueueMessageReady.java

```diff
@@ -19,12 +20,13 @@ public QueueMessageReady(String flightId) {
   }
 
   @Override
-  public void process(Stairway stairway) throws InterruptedException {
+  public void process(Object dispatchContext) throws InterruptedException {
+    StairwayImpl stairwayImpl = (StairwayImpl) dispatchContext;
     try {
-      boolean resumed = stairway.resume(flightId);
+      boolean resumed = stairwayImpl.resume(flightId);
       logger.info(
           "Stairway "
-              + stairway.getStairwayName()
+              + stairwayImpl.getStairwayName()
               + (resumed ? " resumed flight: " : " did not resume flight: ")
               + flightId);
     } catch (DatabaseOperationException ex) {
```

Comment:
should this propagate false back to the QueueProccessFunction if we get an exception? Do we know if the flightId has been resumed or not?

Reply:
I added some documentation and made it return true. It is the not found case.

Reply:
thanks, the `return false` on catch was what I was asking about.